public/app/routes/routes.tsx

```diff
@@ -309,6 +309,7 @@ export function getAppRoutes(): RouteDescriptor[] {
     // ADMIN
     {
       path: '/admin/authentication',
+      roles: () => contextSrv.evaluatePermission(() => ['Admin'], [AccessControlAction.SettingsRead]),
```

Comment:
This is minor but I think this technically blocks the route for the `Server Admin+Viewer` users when RBAC is disabled.

Reply:
Do we still need to support this despite RBAC will be enabled by default in G10?

Reply:
What Misi said - I think it's safe to ignore the legacy access control, but having `Admin` here is a safe option in case we for some reason revert to the legacy AC.

Reply:
On second thoughts, there's no harm in extending it to Server Admin 🤔 So I'll do that.